content-review/impl/service/src/main/java/org/sakaiproject/contentreview/service/ContentReviewQueueServiceImpl.java

```diff
@@ -52,7 +52,10 @@ public void queueContent(Integer providerId, String userId, String siteId, Strin
 		Objects.requireNonNull(siteId, "siteId cannot be null");
 		Objects.requireNonNull(taskId, "taskId cannot be null");
 		Objects.requireNonNull(content, "content cannot be null");
-				
+		
+		boolean errorsPresent = false;
+		StringBuilder errors = new StringBuilder();
```

Comment:
can't you just check if errors is empty instead of having a separate boolean?

Reply:
That would work, yes, but StringBuilder doesn't have an isEmpty() or anything though, so I'd end up doing something like StringUtils.isEmpty(errors.toString())
I figured a well named boolean serves best. Let me know your preference

Reply:
errors.Length > 0 

I think general best practice is to minimize database fields and variables to minimum necessary. It's almost always better to use one column in a database table instead of two columns (e.g., a boolean and a value)

Reply:
Refactored accordingly

Reply:
sorry guess you have to toString() to check it

Reply:
Where's that, line 83? I've tested a StringBuilder's .length(), it seems to work fine

Reply:
Okay, awesome. I was looking at javadocs and wasn't seeing it